Web/Edubase.Web.UI/Areas/Establishments/Models/TabEditPolicy.cs

```diff
@@ -22,6 +23,11 @@ public class TabEditPolicy
         public TabEditPolicy(EstablishmentModel model, EstablishmentDisplayEditPolicy policy, IPrincipal principal)
         {
             Governance = !(principal.IsInRole(EdubaseRoles.ESTABLISHMENT) && model.StatusId.OneOfThese(ES.Closed));
+            Location = new[]
+            {
+                policy.RSCRegionId, policy.GovernmentOfficeRegionId, policy.AdministrativeDistrictId, policy.AdministrativeWardId, policy.ParliamentaryConstituencyId,
+                policy.UrbanRuralId, policy.GSSLAId, policy.Easting, policy.Northing, policy.MSOAId, policy.LSOAId
+            }.Any(x => x == true);
```

Comment:
This list appears to be consistent with the fields listed on the location edit tab.

Perhaps we could add a sign-posting comment to make it easier for future folks to cross-reference between the files / understand the rationale for why these fields specifically?

[Edubase.Web.UI.Areas.Establishments.Controllers.EstablishmentController.EditLocation](https://github.com/DFE-Digital/get-information-about-schools/blob/6904d3f607458c307cff7af5c68e48e451414eac/Web/Edubase.Web.UI/Areas/Establishments/Controllers/EstablishmentController.cs#L526-L555)